tests/unit/test__helpers.py

```diff
@@ -544,12 +544,12 @@ def test_with_bytes(self):
         self.assertEqual(self._call_fut(value), value)
 
     def test_with_unicode(self):
-        value = u"string-val"
+        value = "string-val"
```

Comment:
There are also a few remaining occurrences of these prefixes in `docs/conf.py`.

Reply:
Templated file.

Reply:
Update upstream pending: https://github.com/googleapis/synthtool/pull/1119